### PR TITLE
Poke

### PR DESCRIPTION
**Pull request by EndymionJkb**
Changes proposed in this pull request:
- More explicit description of gradual update in About (start and end times, including block numbers)
- Try to avoid errors by disabling poke when we know it will fail (e.g., too early)

### CODE REVIEW

File: src/views/Pool/About.vue [hunk 1]
```diff
@@ -45,7 +45,14 @@
             )}`
           "
         />
-        <div v-else v-text="`${$t('ongoingUpdateWarning')} ${endTime}`" />
+        <div
+          v-else
+          v-html="
+            `${$t('ongoingUpdateFrom')} ${startTime} (${
```
Comment (bonustrack) [inline src/views/Pool/About.vue:51]: It's better if we use the function params populate translation message, something like this:
```js
$t('ongoingUpdateFrom', { startTime, startBlock: bPool.metadata.startBlock, endTime, ... })
```
In en-US.json we can use `There is a gradual weights update underway<br> from {startTime} {startBlock}...`

File: src/views/Pool/About.vue [hunk 2]
```diff
@@ -45,7 +45,14 @@
...
+              this.bPool.metadata.startBlock
+            }) <br>${$t('until')} ${endTime} (${this.bPool.metadata.endBlock})`
```
Comment (bonustrack) [inline src/views/Pool/About.vue:53]: We don't need to use `this` anywhere in the view, `bPool` will work same as `this.bPool`.

File: src/views/Pool/About.vue [hunk 3]
```diff
@@ -195,21 +202,37 @@ export default {
...
+        {
+          month: 'long',
+          day: 'numeric',
+          hour: '2-digit',
+          minute: '2-digit'
+        }
+      );
+    },
+    endTime() {
+      return this.blockDate(this.bPool.metadata.endBlock).toLocaleString(
```
Comment (bonustrack) [inline src/views/Pool/About.vue:217]: Instead of "toLocaleString" we could use the method `$d(date, 'long')` from `vue-i18n`

**Review by bonustrack — COMMENTED** [review]
Looks good! I've added few suggestions

File: src/views/Pool/About.vue (hunk 1 above)
Comment (EndymionJkb) [reply src/views/Pool/About.vue]: Done

File: src/views/Pool/About.vue (hunk 3 above)
Comment (EndymionJkb) [reply src/views/Pool/About.vue]: The pre-formatted dates don't look like how I want. I could move it to the blockDate function though, to not duplicate code.

**Review by bonustrack — APPROVED** [review]
Looking good!

**Pull request merged**